From faba92b339299826af8f602e621a3939422954ac Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Tue, 28 Aug 2018 19:08:05 +0300
Subject: [PATCH 0853/1200] defconfig: db-88f6820-gp: Add bubt command to the
 build

Add Marvell bubt command to db-88f6820-gp defconfig

Change-Id: Ic161f9dcb78a5074de59c796fda1a0fea371a87d
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/59565
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Igal Liberman <igall@marvell.com>
---
 configs/db-88f6820-gp_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/db-88f6820-gp_defconfig b/configs/db-88f6820-gp_defconfig
index 6863124c07..c5b7d57522 100644
--- a/configs/db-88f6820-gp_defconfig
+++ b/configs/db-88f6820-gp_defconfig
@@ -36,6 +36,7 @@ CONFIG_CMD_EXT2=y
 CONFIG_CMD_EXT4=y
 CONFIG_CMD_FAT=y
 CONFIG_CMD_FS_GENERIC=y
+CONFIG_CMD_MVEBU_BUBT=y
 CONFIG_EFI_PARTITION=y
 # CONFIG_PARTITION_UUIDS is not set
 # CONFIG_SPL_PARTITION_UUIDS is not set
-- 
2.22.0

